Apidaze.SDK/Recordings/Recordings.cs

```diff
@@ -27,30 +29,23 @@ public Stream DownloadRecording(string sourceFileName)
             return new MemoryStream(response);
         }
 
-        public async Task DownloadRecodingToFileAsync(string sourceFileName, string target)
+        public async Task<FileInfo> DownloadRecodingToFileAsync(string sourceFileName, string target)
         {
             var restRequest = DownloadRequest(sourceFileName);
             var response = await Client.ExecuteTaskAsync(restRequest);
 
-            if (response.StatusCode != HttpStatusCode.OK)
-            {
-                throw new InvalidOperationException(response.ErrorMessage);
-            }
+            if (response.StatusCode != HttpStatusCode.OK) throw new InvalidOperationException(response.ErrorMessage);
 
-            SaveFileToFolder(sourceFileName, target, response);
+            return SaveFileToFolder(sourceFileName, target, response);
```

Comment:
Could we get rid of duplicate code in async vs sync methods? Throughout the whole SDK